lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -234,7 +234,7 @@ protected function transformCollection($items, $method = 'transform') {
    */
   protected function transformCampaign($data) {
     $output = array(
-      'id' => $data->id ?: $data->nid,
+      'id' => isset($data->id) ? $data->id : $data->nid,
```

Comment:
@aaronschachter changed to use the longer form because it was throwing a notice that the property was undefined... I thought it was registering as `null` but I was incorrect. So need the longer form to check if set first!